### PR TITLE
Hw1.3

### PR DESCRIPTION
**Pull request by MarberyUA**

### CODE REVIEW

File: src/main/java/core/basesyntax/BinaryString.java [hunk 1]
```diff
@@ -7,6 +7,17 @@
      * а возвращает String с представлением этого числа в двоичном виде.
      */
     public String toBinaryString(int value) {
-        return null;
+        if (value == 0) {
+            return "0";
+        }
+
```
Comment (dmytro-hryhoruk) [inline src/main/java/core/basesyntax/BinaryString.java]:
```suggestion
```

File: src/main/java/core/basesyntax/BinaryString.java [hunk 2]
```diff
@@ -7,6 +7,17 @@
...
+        int b;
```
Comment (dmytro-hryhoruk) [inline src/main/java/core/basesyntax/BinaryString.java]: do we need this variable?

File: src/main/java/core/basesyntax/BinaryString.java [hunk 3]
```diff
@@ -7,6 +7,17 @@
...
+        StringBuilder temp = new StringBuilder("");
+        while (value != 0) {
+            b = value % 2;
+            temp.insert(0,b);
```
Comment (dmytro-hryhoruk) [inline src/main/java/core/basesyntax/BinaryString.java]: try to use append()

Comment (dmytro-hryhoruk) [reply src/main/java/core/basesyntax/BinaryString.java]: it'll makes your code more understandable

File: src/main/java/core/basesyntax/BinaryString.java (hunk 2 above)
Comment (MarberyUA) [reply src/main/java/core/basesyntax/BinaryString.java]: Yup, we do not need that)

File: src/main/java/core/basesyntax/BinaryString.java (hunk 1 above)
Comment (MarberyUA) [reply src/main/java/core/basesyntax/BinaryString.java]: Deleted, sorry for extra spaces

File: src/main/java/core/basesyntax/BinaryString.java (hunk 3 above)
Comment (MarberyUA) [reply src/main/java/core/basesyntax/BinaryString.java]: Done, thanks)

File: src/main/java/core/basesyntax/BinaryString.java [hunk 4]
```diff
@@ -7,6 +7,15 @@
...
+        if (value == 0) {
+            return "0";
+        }
+        StringBuilder temp = new StringBuilder("");
+        while (value != 0) {
+            temp.append(value % 2);
+            value = value / 2;
+        }
+        temp.reverse();
+        return temp.toString();
```
Comment (dmytro-hryhoruk) [inline src/main/java/core/basesyntax/BinaryString.java:19]:
```suggestion
        return temp.reverse().toString();
```

**Review by dmytro-hryhoruk — APPROVED** [review]

**Pull request closed without merging**